tests/test_config.py

```diff
@@ -0,0 +1,18 @@
+
+def test_config():
+    import ucp
+    config = ucp.get_config()
+    assert isinstance(config, dict)
+    assert config['UCX_MEMTYPE_CACHE'] == 'n'
+    assert config['UCX_SEG_SIZE'] == '8K'
+
+
+def test_set_config():
+    import os
+    os.environ['UCX_SEG_SIZE'] = '2M'
+    import ucp
+    ucp.public_api._ctx = None
```

Comment:
Not exactly sure why but I needed to set the context to `None` (even though the imports are local) to register the env var change ?

Reply:
Nice idea!
It is because in Python a module is only unloaded at termination. 
I have added `ucp.reset()` to make it more user friendly.